Lib/test/test_json/test_tool.py

```diff
@@ -61,12 +61,11 @@ class TestTool(unittest.TestCase):
     """)
 
     def test_stdin_stdout(self):
-        with subprocess.Popen(
-                (sys.executable, '-m', 'json.tool'),
-                stdin=subprocess.PIPE, stdout=subprocess.PIPE) as proc:
+        args = sys.executable, '-m', 'json.tool'
+        with Popen(args, stdin=PIPE, stdout=PIPE, stderr=PIPE) as proc:
             out, err = proc.communicate(self.data.encode())
         self.assertEqual(out.splitlines(), self.expect.encode().splitlines())
-        self.assertEqual(err, None)
+        self.assertEqual(err, b'')
```

Comment:
> By the way, `--no-ensure-ascii` and `--indent/--no-indent` are two separate enhancements so there should be two pull requests.

@berkerpeksag will keep this PR for `--no-ensure-ascii` and open a new one for `--indent/--no-indent`. I think it would make sense to merge the indent PR first, as it's closer to complete.